LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 QPixel Contributors
+Copyright (c) 2017 Fabrizio Bertoglio
```

Comment:
**Unsolicitedly moving copyright to you exclusively is illegal**. The 'QPixel Contributors' would have to move the copyright to you themselves, and since that's multiple persons, well...